working-w-copilot/copilot-in-projects-refactoring.md

```diff
@@ -12,13 +12,13 @@ Our goals for this lesson are to:
 
 Writing new code with Copilot is great, but a significant portion of software development involves refactoring and updating existing code. We got a little taste of updating code with the `Book` class, but we're going to revisit a scenario that we've seen before—refactoring the `validate_book` function into a more generic function that we can use for any model.
 
-For this section of the lesson, we're going to check out a different `hello-books` branch, [`07b-from-dict-refactor`](https://github.com/AdaGold/hello-books-api/tree/07b-from-dict-refactor), as the starting point.
+For this section of the lesson, we're going to check out a different `hello-books` branch, [`07b-to-dict-refactor`](https://github.com/AdaGold/hello-books-api/tree/07b-to-dict-refactor), as the starting point.
```

Comment:
Oh no. After we undo the refactor order swap at sme point in the future, this will will need to go back to what was there before. 😅

I added a subtack to that swap back ticket to revist this: https://app.asana.com/0/1201700438643002/1208793757696414/f